docs/computing/containers/run-existing.md

```diff
@@ -0,0 +1,49 @@
+# Running existing containers
+
+Puhti supports running [Singularity](https://sylabs.io/singularity/) containers.  The easiest option is to build containers from existing Docker images.
+
+
+## Building containers from existing Docker images
+
+Singularity containers are typically built from Docker images, and the easiest way to get started is to use a pre-built Docker image.  For example Nvidia has a library of container images for different GPU-enabled applications in their [Nvidia GPU cloud (NGC)](https://ngc.nvidia.com/).
+
+Here is an example how to build a Singularity image from Nvidia's PyTorch Docker image on a Puhti compute node:
+
+```bash
+# We'll reserve 100GB NVME for fast temporary disk space
+srun -A <project> --gres=nvme:100 -t 1:00:00 --mem=16G --pty $SHELL
+
+# Let's use the NVME space as $TMPDIR
+export TMPDIR=$LOCAL_SCRATCH
```

Comment:
Would using `SINGULARITY_TMPDIR` be better, as stated in <https://sylabs.io/guides/3.5/user-guide/build_env.html?highlight=singularity_tmpdir>

Also should we instruct to set the cache dir with  `SINGULARITY_CACHEDIR`? so that the home folder is not filled.

Reply:
Agreed, fixed in latest commit. (Now setting both SINGULARITY_TMPDIR and SINGULARITY_CACHEDIR)